### ISSUE TITLE
Issues with the plugin

### ISSUE THREAD

**Issue by nezam05** [rcarabelli/Mautic-Zender-Plugin#22]
First, Thank you for the plugin. Excellent open source work.

Secondly, The plugin functionality doesn't conform with what the README promises. 

1. On activation, It doesn't show any Features tab
2. it says ``Zender API URL: The base URL for the Zender API (up to the /api part)``, whereas it requires full url _domain/api/send/whatsapp_ to send whatsapp message.
3. It says, ``Six new fields are added to the leads table to save ``, and none are created.

**Comment by rcarabelli** [comment]
You downloaded the last release?
https://github.com/rcarabelli/Mautic-Zender-Plugin/releases/tag/v.1.1.0 

If this is the case, let me know to correct it.

Thanks

**Comment by nezam05** [comment]
Not from Tag, but downloaded the repo zip file. I guess these two are the same. Just inspected the code.
https://github.com/rcarabelli/Mautic-Zender-Plugin/archive/refs/heads/main.zip

**Comment by rcarabelli** [comment]
Do you want to check it with me in a meet, zoom or other way to test it together?

**Comment by nezam05** [comment]
yes. I'll setup a mautic instance with the plugin. can you share an email so that I can share the details?

**Comment by rcarabelli** [comment]
You can find me at mautic slack as rcarabelli or via my help desk email in the readme file: [requests@7catstudio.com](mailto:requests@7catstudio.com)

I answer daily.
